admin/securedrop_admin/__init__.py

```diff
@@ -512,7 +512,7 @@ def backup_securedrop(args):
         'ansible-playbook',
         os.path.join(args.ansible_path, 'securedrop-backup.yml'),
     ]
-    subprocess.check_call(ansible_cmd, cwd=args.ansible_path)
+    return subprocess.check_call(ansible_cmd, cwd=args.ansible_path)
```

Comment:
- The logic here makes sense if this returns 0, as a `sys.exit(0)` will occur in `main` ✅ 
- If the command returns non-zero, then this will raise a `subprocess.CalledProcessError`, which gets caught in `main` and always produces a `SystemExit` with return code 1 regardless of the return code on `subprocess.CalledProcessError`, so I think we want to refine this further. I understand that might be a bit much for pre-release changes, but let's at least file a followup to address?

Reply:
Actually I'll retarget the existing issue since we cleaned up enough for 0.7.0